### PR TITLE
Dylann

### PR DESCRIPTION
**Pull request by dylannalcock**
Finished the main menu, need to link start button to game scene still.

### CODE REVIEW

**Comment by JimmyC7834** [comment]
@dylannalcock there's no menu in the menu scene. The scene is empty.

File: Assets/UI/Menu Scripts/MenuScript.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
```
Comment (JimmyC7834) [inline Assets/UI/Menu Scripts/MenuScript.cs:1]: What is this folder and script for? they are all empty

File: Assets/UI/MenuParticles/Particle/particle1.mat [hunk 2]
```diff
@@ -0,0 +1,98 @@
+%YAML 1.1
```
Comment (JimmyC7834) [inline Assets/UI/MenuParticles/Particle/particle1.mat:1]: What is the use of the particle thing? What are they used for? I don't see any use cases in your changes

**Comment by JimmyC7834** [comment]
@dylannalcock  Please also put all the script under the script folder, prefab under the prefab folder, etc.

**Review by Teru5244 — CHANGES_REQUESTED** [review]
In my perspective your PR is incomplete or even empty, please finish the implementation and address the problems proposed by Jimmy above before PR them.

**Review by JimmyC7834 — APPROVED** [review]

**Review by Teru5244 — APPROVED** [review]

File: Assets/Scripts/MenuScript.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuScript : MonoBehaviour
+{
+
+    public string firstLevel;
```
Comment (JimmyC7834) [inline Assets/Scripts/MenuScript.cs:9]: We should always make private variables
This is not following the naming scheme please goto read the C# style guide in our repo's root folder

File: Assets/Scripts/MenuScript.cs [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    // Start is called before the first frame update
+    void Start()
```
Comment (JimmyC7834) [inline Assets/Scripts/MenuScript.cs:11]: Remove empty functions

File: Assets/Scripts/MenuScript.cs [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
```
Comment (JimmyC7834) [inline Assets/Scripts/MenuScript.cs:17]: Remove empty functions

File: Assets/Scripts/MenuScript.cs [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        
+    }
+
+    public void PlayGame() {
```
Comment (JimmyC7834) [inline Assets/Scripts/MenuScript.cs:22]: Not following style guide, we do it like this:
```
public void PlayGame()
{
...
}
```

File: Assets/Scripts/MenuScript.cs [hunk 7]
```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuScript : MonoBehaviour
```
Comment (JimmyC7834) [inline Assets/Scripts/MenuScript.cs:6]: All of our script should be in the `Game` namespace

**Review by JimmyC7834 — COMMENTED** [review]
@dylannalcock please review your mistakes and don't it them next time

**Pull request merged**